src/components/Header/Header.jsx

```diff
@@ -21,29 +21,38 @@ import BurgerMenu from '../BurgerMenu';
 const tablet = 864;
 const mobile = 425;
 
-const navbarPages = [
 // TODO Explorer
 // TODO Matches
-  {
-    name: strings.header_heroes,
-    path: '/heroes',
-  }, {
-    name: strings.header_distributions,
-    path: '/distributions',
-  }, {
-    name: strings.header_ingame,
-    sponsored: true,
-    external: true,
-    path: 'https://dota2.becomethegamer.com/yasp',
-  }];
+const navbarPages = [{
+  name: strings.header_heroes,
+  path: '/heroes',
+}, {
+  name: strings.header_distributions,
+  path: '/distributions',
+}, {
+  name: strings.header_ingame,
+  sponsored: true,
+  external: true,
+  path: '//dota2.becomethegamer.com/yasp',
+}];
+
+const navbarBug = {
+  name: strings.app_report_bug,
+  external: true,
+  path: '//github.com/odota/ui/issues',
+};
+
+const navbarPagesLength = navbarPages.length;
 
 const LogoGroup = ({ width }) => {
-  if (width <= mobile) {
-    navbarPages.push({
-      name: strings.app_report_bug,
-      external: true,
-      path: 'https://github.com/odota/ui/issues',
-    });
+  if (width < mobile && navbarPagesLength === navbarPages.length) {
+    navbarPages.push(navbarBug);
+  }
+
+  // If `bug` item in navbar and it's becoming wider, remove `bug` from navbarPages
```

Comment:
Actually how about:
* Define a new const that's a copy of navbarPages in this function's scope
* Push report bug onto it if width is < mobile
* That way we avoid mutating navbarPages and having to do the check, as well as avoiding the need to splice